docs/src/contributing/development-environment.md

```diff
@@ -1,280 +1,266 @@
+Install Docker Machine according to the normal installation instructions, then use it to create a new VM:
 
-    $ docker-machine create --driver virtualbox --virtualbox-disk-size=100000 \
-      --engine-insecure-registry=192.168.0.0/16 deis-registry
+```
+$ docker-machine create deis-docker \
+    --driver=virtualbox \
+    --virtualbox-disk-size=100000 \
+    --engine-insecure-registry=192.168.0.0/16
+```
 
-This will create a new virtual machine named `deis-registry` that will take up as much as 100,000 MB of disk space. Registries tend to be large, so allocating a big disk is a good idea.
+This will create a new virtual machine named `deis-docker` that will take up as much as 100,000 MB of disk space. The images you build may be large, so allocating a big disk is a good idea.
 
-Once the deis-registry machine exists, source its values into your environment so your docker client knows how to use the new machine.
+Once the `deis-docker` machine exists, source its values into your environment so your docker client knows how to use the new machine. You may even choose to add this to your bash profile or similar.
 
-    $ eval "$(docker-machine env deis-registry)"
+```
+$ eval "$(docker-machine env docker-deis)"
+```
 
 !!! note
-    Because the registry that we create will not have a valid SSL certificate, we run the local registry as an insecure (HTTP, not HTTPS) registry. Each time Docker Machine reboots, the registry will get a new IP address somewhere in the `192.168.0.0/16` range. We must declare that explicitly when configuring Docker Machine.
-
-At this point, our `deis-registry` VM can now serve as a registry for Deis' Docker images. Later we will return to this.
+    In subsequent steps, you may run a Docker registry within the `deis-docker` VM. Such a registry will not have a valid SSL certificate and will use HTTP instead of HTTPS. Such registries are implicitly untrusted by the Docker server (which is also running on the `deis-docker` VM).  In order for the Docker server to trust the insecure registry, `deis-docker` is explicitly created to trust all registries in the `192.168.0.0/16` range - the range in which the VM (and therefore the registry) will exist.  This will effectively permit Docker pulls and pushes to such a registry.
 
-## Fork the Deis Repository
+## Fork the Repository
 
-Once the prerequisites have been met, we can begin to work with Deis.
+Once the prerequisites have been met, we can begin to work with Deis components.
 
-To get Deis running for development, first [fork the Deis repository][fork], then clone your fork of the repository. Since Deis is predominantly written in Go, the best place to put it is in `$GOPATH/src/github.com/deis/`.
+Begin at Github by forking whichever Deis project you would like to contriubte to, then clone that fork locally.  Since Deis is predominantly written in Go, the best place to put it is in `$GOPATH/src/github.com/deis/`.
```

Comment:
Yes.  That sounds better.  That bit of verbiage carried over from the old doc, as you can see, but it's worth fixing now.